tools/codegen/generators.ts

```diff
@@ -941,7 +937,7 @@ import { symbols } from "./_symbols.ts";
   lines.push(...imports);
   lines.push("");
 
-  lines.push(`let _library: DynamicLibrary<typeof symbols> = null!;`);
+  lines.push(`let _library: Deno.DynamicLibrary<typeof symbols> = null!;`);
```

Comment:
I actually don't want this change. I've been putting as much effort as I can into having `deno` specific code only in `src/deno` as I would like to long term have bindings to other runtimes such as `bun` or `web`.

Reply:
Generated code should only be deployed to deno platform, otherwise it is impossible to reuse deno style FFI on other platform

Reply:
we can handle cross platform ffi at code generator, rather than generated code

Reply:
@smack0007 these change is removed